helperAPI.py

```diff
@@ -696,62 +727,87 @@ def maskString(string):
     return masked
 
 
-async def printHoldings(
-    botObj: commands.Bot,
-    expected_user_id: int,
-    brokerObj: Brokerage,
-    loop=None,
-    mask=True,
-):
-    # Fetch the user object using their ID
-    try:
-        user = await botObj.fetch_user(expected_user_id)
-        if not user:
-            raise ValueError(f"Could not find user with ID {expected_user_id}")
-    except Exception as e:
-        printAndDiscord(str(e), loop)
-        return None
-
-    # Prepare embed for Discord message
-    embed = discord.Embed(title=f"{brokerObj.get_name()} Holdings", color=3447003)
-
+def printHoldings(brokerObj: Brokerage, loop=None, mask=True):
     # Helper function for holdings formatting
-
```

Comment:
don't change our implementation for this, revert please

Reply:
revert all code and just added. thx, @sudodeo